util/opensslwrap.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
 
-HERE="`echo $0 | sed -e 's|[^/]*$||'`"
-OPENSSL="${HERE}../apps/openssl"
+HERE="`echo $0 | sed -e 's|[^/]*$||'`.."
+OPENSSL="${HERE}/apps/openssl"
 
-if [ -d "${HERE}../engines" -a "x$OPENSSL_ENGINES" = "x" ]; then
-	OPENSSL_ENGINES="${HERE}../engines"; export OPENSSL_ENGINES
+if [ -z "$OPENSSL_ENGINES" ] && [ -d "${HERE}/engines" ] ; then
```

Comment:
What's wrong with `-a`?  Is there a reason you want to create two process when one can be enough?

Reply:
Isn't it builtin anyway? The reason I changed it can be found in the TEST(1) manpage:

       NOTE:  Binary  -a  and -o are inherently ambiguous.  Use 'test EXPR1 &&
       test EXPR2' or 'test EXPR1 || test EXPR2' instead.

Reply:
IIRC, the `-a` evaluates both expressions, no?